MapboxDirections/MBRouteStep.swift

```diff
@@ -623,6 +626,8 @@ open class RouteStep: NSObject, NSSecureCoding {
         destinations = decoder.decodeObject(of: [NSArray.self, NSString.self], forKey: "destinations") as? [String]
         
         intersections = decoder.decodeObject(of: [NSArray.self, Intersection.self], forKey: "intersections") as? [Intersection]
+        
+        instructionsSpokenAlongStep = decoder.decodeObject(of: [NSArray.self, VoiceInstruction.self], forKey: "voiceInstructions") as? [VoiceInstruction]
```

Comment:
@1ec5 how do you feel about moving away from the word `spoken` in favor of `voice`? I find myself naming variables voice a lot in https://github.com/mapbox/mapbox-navigation-ios/pull/614/files

Reply:
“Voice” is a noun, not an adjective. In both AVFoundation and Polly, a “voice” refers is a style of speech output that you can choose, e.g. “Joanna” versus “Giorgio”; it doesn’t refer to the act of speaking. Along these lines, “SpokenInstructions” would be preferable to “VoiceInstructions”, and if we name a method that does the speaking, it should be called “speak”, not “voice”.